### PR TITLE
hw 1 added solution to matchResult

### PR DESCRIPTION
**Pull request by maks45**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        return result1 == bet1 && result2 == bet2 ? 2
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: Good work! move '?' to a next line)

Comment (maks45) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,12 @@
...
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        return result1 == bet1 && result2 == bet2
+                ? 2
+                : (result1 == result2 && bet1 == bet2)
+                || (result1 > result2 && bet1 > bet2)
+                || (result1 < result2 && bet1 < bet2)
+                ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        return result1 == bet1 && result2 == bet2
                ? 2 : (result1 == result2 && bet1 == bet2)
                || (result1 > result2 && bet1 > bet2)
                || (result1 < result2 && bet1 < bet2)
                ? 1 : 0;
```

Comment (maks45) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 3]
```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java:18]:
```suggestion
```

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**